Kernel/Syscalls/sysconf.cpp

```diff
@@ -37,6 +38,10 @@ ErrorOr<FlatPtr> Process::sys$sysconf(int name)
         return Process::max_arguments_size;
     case _SC_IOV_MAX:
         return IOV_MAX;
+    case _SC_PHYS_PAGES: {
+        auto system_memory = MM.get_system_memory_info();
```

Comment:
(why not to add a separate one liner? 😅)

Reply:
Sorry, I'm no C/C++ developer. Can you elaborate what you mean with "add a seperate oneliner"? I copied this code from another place 😅

Reply:
I understood now. I can remove an unesessary variable. Force pushed the branch.